workflows/differentialabundance.nf

```diff
@@ -603,6 +604,34 @@ workflow DIFFERENTIALABUNDANCE {
     ch_versions = ch_versions
         .mix(DIFFERENTIAL_FUNCTIONAL_ENRICHMENT.out.versions)
 
+        // Run DECOUPLER
+    if (params.decoupler_run && params.gtf){
+        log.warn "Decoupler is only supported for human or mouse datasets. Please ensure your organism is compatible before enabling this module."
```

Comment:
I'm not even sure we need that since the decoupler network is a user-provided parameter. It's up to the user to make sure it has compatible gene identifiers. 

decoupler (the method) has no limiation on the organism. It's only that Dorothea or progeny (i.e. the references) are only available for human and mouse. But any geneset can work with decoupler.